app/src/main/res/menu/menu_settings.xml

```diff
@@ -1,7 +1,7 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
       xmlns:app="http://schemas.android.com/apk/res-auto"
       xmlns:tools="http://schemas.android.com/tools"
-      tools:context="uk.echosoft.garageopener.SettingsActivity">
+      tools:context=".SettingsActivity">
```

Comment:
These XML files are inconsistent, the package structure is included in some and removed from others, can you explain why?